src/Symfony/Component/Mime/Part/DataPart.php

```diff
@@ -26,10 +26,13 @@ class DataPart extends TextPart
     private $mediaType;
     private $cid;
 
+    /** @internal */
```

Comment:
This looks not needed for a private one

Reply:
Oh i thought i do it like the protected `$_parent` but for a private that's right - no needed. I'll remove it in a commit to also see if the tests break again (might not be my problem?).

Reply:
```suggestion
```